public/index.html

```
@@ -18,172 +19,209 @@
+  <!-- ── Activity Browser ───────────────────────────────────────────────── -->
+  <main class="max-w-7xl mx-auto px-4 sm:px-6 lg:px-8 py-12" id="activities">
 
-  <!-- Activity grid -->
-  <div id="activity-grid" class="grid grid-cols-1 sm:grid-cols-2 lg:grid-cols-3 gap-6">
-    <div class="bg-white rounded-2xl shadow-sm p-6 animate-pulse h-52"></div>
-    <div class="bg-white rounded-2xl shadow-sm p-6 animate-pulse h-52"></div>
-    <div class="bg-white rounded-2xl shadow-sm p-6 animate-pulse h-52"></div>
-  </div>
+    <!-- Search + Filters -->
+    <div class="flex flex-col sm:flex-row gap-3 mb-4">
+      <input id="search" type="text" placeholder="🔍  Search activities…"
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)